user/preprocess/ascii-mesh-particles.md

```diff
@@ -64,11 +64,11 @@ An example of a 3D mesh comprising of two cells and twelve nodes is shown below:
 Velocity constraints at each node can be specified in the following format:
 
 ```
-n_0     d_1     v_1
-n_1     d_0     v_0
+n_0     d_0     v_0
```

Comment:
Is this still valid?

Reply:
Now that we will have this option, I think it is still valid.